handler/bot.go

```diff
@@ -62,3 +65,19 @@ func (b *Bot) Process() error {
 
 	return nil
 }
+
+func constructGroupedPrMessage(pullRequests []entity.PullRequest) string {
+	repoMap := make(map[string][]entity.PullRequest)
+	for _, pr := range pullRequests {
+		repoMap[string(pr.Repository.Name)] = append(repoMap[string(pr.Repository.Name)], pr)
```

Comment:
Having this should be enough:
```suggestion
		repoMap[string(pr.Repository.Name)] = pr
```

Reply:
this part is appending `pr` to the list of PRs (`repoMap` value is `[]entity.PullRequest`), not just assigning it

Reply:
you're right, didn't notice that.